examples/todoapp/lib/base/di/todoapp_with_dependencies.dart

```diff
@@ -165,16 +171,16 @@ class TodoappWithDependencies extends StatelessWidget {
           ),
         ),
         Provider<TodoActionsService>(
-          create: (context) => TodoActionsService(context.read()),
-        )
+          create: (context) => TodoActionsService(
+            context.read(),
+          ),
+        ),
       ];
 
   List<SingleChildWidget> get _blocs => [
         Provider<RouterBlocType>(
-          create: (context) => RouterBloc(
-            router: context.read<AppRouter>().router,
-            permissionsService: context.read(),
-          ),
+          create: (context) =>
+              RouterBloc(router: context.read<AppRouter>().router),
```

Comment:
Why the router does not depend on the permissions?

Reply:
Fixed